src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -34,6 +34,7 @@
       <OutputItemType>Content</OutputItemType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Yes, the <OSGroup> in project-to-project references needs to be $(OSEnvironment)